app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -106,6 +106,23 @@ examples:
           value: "green"
         - label: "Blue"
           value: "blue"
+  with_page_header_and_caption:
+    description: |
+      If a caption text is provided with a page heading, it will be displayed above the heading.
+      A caption can only be used with a page heading. If a heading is not provided the caption will not render.
+      The pattern is used across GOV.UK to show a high-level section that this page question falls into.
```

Comment:
I don't know if you want to or not, but in order to have actual line breaks rendered in the documentation you need to put an empty line here between lines. Up to you.

Reply:
Ah i see, humm... I think I'd mostly put those in to stop the lines getting too long.
I think that's what's going on with the other descriptions above..?

Might leave it as is for now, cheers for flagging!